autoscheduler/scraper/serializers.py

```diff
@@ -36,3 +37,56 @@ def get_meetings(self, obj): # pylint: disable=no-self-use
             'end_time': meeting.end_time,
             'type': meeting.meeting_type,
         } for meeting in meetings]
+
+class TermSerializer(serializers.ModelSerializer):
+    """ Serializes a term into an object with information needed by /api/terms """
+    desc = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Department
+        fields = ['term', 'desc']
+
+
+    def get_desc(self, obj):
+        """ Uses term field to generate description for the term in the
+            form "Fall - College Station" format"""
+
+        def season_num_to_string(season_num):
+            """" Converts int representing season in 'term' field to a string to
+                use in get_term """
+            # put all translations here. Possibly inaccurate atm.
+            seasons = {
+                1: "Spring",
+                2: "Summer",
+                3: "Fall",
+                4: "Winter",
+            }
+            return seasons.get(season_num, "NO SEASON")
+
+        def campus_num_to_string(campus_num):
+            """" Converts int representing campus in 'term' field to a string to
+                use in get_term """
+            # put all translations here. Possibly inaccurate atm.
+            campus = {
+                1: "College Station",
+            }
```

Comment:
See [this comment](https://github.com/aggie-coding-club/Rev-Registration/issues/115#issuecomment-576746043) for a more robust list of campuses, and add them to this dict